app/controllers/application_controller.rb

```diff
@@ -1,5 +1,5 @@
 class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
-  protect_from_forgery with: :exception
+  # protect_from_forgery with: :exception # nope, we've decided we want to be unsafe
```

Comment:
Is this really the comment you want?

It isn't that you want it to be unsafe. It is that you are doing something different than what rails was originally set up for. Security is still important. But this particular aspect is not relevant for what you are doing. Stating that you want to be unsafe means that there is a way to secure things and do what you want to do and you are willfully choosing not to do it... in which case, why not just do it?